tests/zenko_e2e/backbeat/Using.md

```diff
@@ -10,34 +10,44 @@
    ```
 
 2. Create an account using Orbit.
-3. Export the access key and secret key of that account:
+3. Export the access key and secret key of that account (for example, in
```

Comment:
I think 3 and 7 can easily be combined into one step.  Something like `Secrets can be added to the secret.env file and sourced into your test environment`

Reply:
Okay, I've moved them as two sequential steps. I've kept them as distinct numbers though to maintain the indentation.